frame/support/src/storage/bounded_btree_map.rs

```diff
@@ -271,6 +274,15 @@ where
 	}
 }
 
+impl<K, V, S> DerefMut for BoundedBTreeMap<K, V, S>
+where
+	K: Ord,
+{
+	fn deref_mut(&mut self) -> &mut Self::Target {
+		&mut self.0
+	}
+}
+
```

Comment:
I needed to implement this so I could do `sub_pools.with_era.entry(current_era).or_insert_with(|| UnbondPool::default());`

Reply:
Same comment as with the other PR, need to be added manually unfortunately.